File: problemas/1000/1013.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = int.parse(entrada[0]);
  final b = int.parse(entrada[1]);
  final c = int.parse(entrada[2]);
  final d = (a + b + (a - b).abs()) ~/ 2;
  final maior = (c + d + (c - d).abs()) ~/ 2;
  stdout.writeln('$maior eh o maior');
}

File: problemas/1000/1013.js
const [a, b, c] = require('fs').readFileSync('/dev/stdin', 'utf-8').split(' ').map(x => Number(x));
const d = (a + b + Math.abs(a - b)) / 2;
const maior = (c + d + Math.abs(c - d)) / 2;

console.log(`${maior} eh o maior`);